src/memex/storage.py

```diff
@@ -108,14 +109,14 @@ def create_annotation(request, data):
             )
 
     # The user must have permission to create an annotation in the group
-    # they've asked to create one in.
-    if data['groupid'] != '__world__':
-        group_principal = 'group:{}'.format(data['groupid'])
-        if group_principal not in request.effective_principals:
-            raise schemas.ValidationError('group: ' +
-                                          _('You may not create annotations '
-                                            'in groups you are not a member '
-                                            'of!'))
+    # they've asked to create one in. If the application didn't configure
+    # a groupfinder we will allow writing this annotation without any
+    # further checks.
+    group = groups.find(request, data['groupid'])
```

Comment:
Ah, now I can reproduce. I tried with a third-party account and it returned that error I posted. Looking into it now